BarcodeParserBuilder/BarcodeParserBuilder.cs

```diff
@@ -35,6 +35,7 @@ public bool TryParse(string? barcodeString, out Barcode? barcode, out string? fe
 
                     //retrieve output parameter and return true
                     barcode = (Barcode?)tryParseParameters[1];
+                    feedback = $"was parsed by {parserBuilder.Name}";
```

Comment:
why are you setting the feedback? don't you know what parser builder parsed it based on the type of barcode youre getting back?